src/components/ClusterPage/index.jsx

```diff
@@ -117,7 +117,13 @@ class ClusterPage extends React.Component {
           <ClustersList newClusterAdded={isAdded} />
         </div>
         <div className="FooterRow">
-          <p>Copyright © 2020 Crane Cloud. All Rights Reserved.</p>
+          <p>
+            Copyright ©
```

Comment:
can we remove the copyright mark? also if u were to use it, i'd advise using a character code (html preferably) instead of inserting it as a symbol

Reply:
@kawerewagaba this has been fixed